mod_ood_proxy/lib/node_proxy.lua

```diff
@@ -36,6 +36,12 @@ function node_proxy_handler(r)
   conn.server = host .. ":" .. port
   conn.uri = uri and (r.args and (uri .. "?" .. r.args) or uri) or r.unparsed_uri
 
+  -- last ditch effort to ensure that the uri is at least something
+  -- because the request-line of an HTTP request _has_ to have something for a URL
+  if conn.uri == '' then
+    conn.uri = '/'
+  end
```

Comment:
I'm guessing at this point it can never be nil. But looking at it again, should it be `if conn.uri == nil or conn.uri == ''`?